ls1intum/Artemis
# Use 0.0 as default value if no credits are present

### Checklist
- [x] Added JUnit test for the bug (that fails when the fix is not present).

### Motivation and Context

Re-evaluate fails for some exercises with manual results because feedback credits are null for some reason.

See Issue: https://sentry.io/organizations/ls1intum/issues/2206895272/

### Description
- Added `Objects.requireNonNullElse` to use `0.0` as default if the feedback credits are null

### Steps for Testing
- Run and check the tests, there is currently no way known to reproduce it on the test server.
